.github/workflows/update-external-docs.yml

```diff
@@ -11,20 +11,20 @@ permissions:
 jobs:
     update-docs:
         runs-on: ubuntu-latest
+        if: github.repository_owner == 'expressjs'
         steps:
-            - uses: actions/checkout@v3
+            - uses: actions/checkout@v4
             - name: Run scripts
               run: bash ./get-contributing.sh && bash ./get-readmes.sh
             - name: Create Pull Request
-              uses: peter-evans/create-pull-request@v5
+              uses: gr2m/create-or-update-pull-request-action@v1
```

Comment:
`peter-evans/create-pull-request` also allows to do team_reviewers (https://github.com/peter-evans/create-pull-request?tab=readme-ov-file#action-inputs). Does `gr2m/create-or-update-pull-request-action` provides any additional feature or improvement? I am just curious.

Reply:
This tool is being changed not because it has other functionalities, but because when executing the workflow, the following error occurred

```
peter-evans/create-pull-request@v5 is not allowed to be used in expressjs/expressjs.com. Actions in this workflow must be: within a repository owned by expressjs, created by GitHub, verified in the GitHub Marketplace, or matching the following: pkgjs/*, gr2m/create-or-update-pull-request-action@*, ulisesgascon/openssf-scorecard-monitor@*.
```

Reply:
Oh! That makes sense... those are rules that we define at the org level :+1: 

Thanks for the clarification